[lambda-http-0.8.2] unresolved imports

When upgrading to `lambda-http-0.8.2`, I get the following error when trying to run `cargo check`. I do not have `lambda_runtime` in my Cargo.toml dependencies. I use the re-exported version of `lambda_runtime` in my code when it is required. (`use lambda_http::lambda_runtime::{run as lambda_run, service_fn, LambdaEvent};`)

```
error[E0432]: unresolved imports `lambda_runtime::FunctionResponse`, `lambda_runtime::MetadataPrelude`, `lambda_runtime::StreamResponse`
  --> /Users/home/.cargo/registry/src/index.crates.io-6f17d22bba15001f/lambda_http-0.8.2/src/streaming.rs:10:56
   |
10 |     self, service_fn, tower, tower::ServiceExt, Error, FunctionResponse, LambdaEvent, MetadataPrelude, Service,
   |                                                        ^^^^^^^^^^^^^^^^               ^^^^^^^^^^^^^^^ no `MetadataPrelude` in the root
   |                                                        |
   |                                                        no `FunctionResponse` in the root
11 |     StreamResponse,
   |     ^^^^^^^^^^^^^^ no `StreamResponse` in the root
```

--- a/lambda-http/Cargo.toml
+++ b/lambda-http/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "lambda_http"
-version = "0.8.2"
+version = "0.8.3"
 authors = [
     "David Calavera <dcalaver@amazon.com>",
     "Harold Sun <sunhua@amazon.com>",
@@ -29,7 +29,7 @@ futures = "0.3"
 http = "0.2"
 http-body = "0.4"
 hyper = "0.14"
-lambda_runtime = { path = "../lambda-runtime", version = "0.8" }
+lambda_runtime = { path = "../lambda-runtime", version = "0.8.3" }
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 serde_urlencoded = "0.7"

